listenbrainz/db/feedback.py

```diff
@@ -0,0 +1,152 @@
+import sqlalchemy
+
+from listenbrainz import db
+
+
+def insert(user_id, recording_msid, score):
+    """ Inserts a feedback record for a user's loved/hated recording into the database.
+        If the record is already present for the user, the score is updated to the new
+        value passed.
+        Args:
+            user_id (int): the row id of the user
```

Comment:
let's start using python type hints in new code, and slowly start migrating other code as we touch it.
In this case, this is as simple as putting these types in the function definition: `def insert(user_id: int, recording_msid: str, score: int):` and removing it from the docstring. Nothing else needed for now

Reply:
Sure